src/ContainerRenameModal.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import {
+    Button,
+    Form, FormGroup,
+    Modal, TextInput
+} from '@patternfly/react-core';
+import cockpit from 'cockpit';
+
+import * as client from './client.js';
+import { ErrorNotification } from './Notification.jsx';
+
+const _ = cockpit.gettext;
+
+const ContainerRenameModal = ({ container, onHide, updateContainerAfterEvent }) => {
+    const [name, setName] = useState(container.Names[0]);
+    const [nameError, setNameError] = useState(null);
+    const [dialogError, setDialogError] = useState(null);
+    const [dialogErrorDetail, setDialogErrorDetail] = useState(null);
+
+    const handleInputChange = (targetName, value) => {
+        if (targetName === "name") {
+            setName(value);
+            if (/^[a-zA-Z0-9][a-zA-Z0-9_\\.-]*$/.test(value)) {
```

Comment:
If it's empty, it shows the wrong message:

![image](https://user-images.githubusercontent.com/67428/166490456-003bdd30-af7d-4908-aee8-474d9b71c549.png)